server/pulp/server/webservices/views/permissions.py

```diff
@@ -184,16 +184,16 @@ def post(self, request):
 
 def _check_invalid_params(params):
```

Comment:
Consider renaming this function to `_validate_params`. I know you didn't choose this name, but in looking at the function it would be clearer and more accurate.